content/techniques/streaming-files.md

```diff
@@ -45,3 +45,26 @@ export class FileController {
   }
 }
 ```
+
+The default content type is `application/octet-stream`, if you need to customize the response you can use `res.set`.
+
+> info **hint** If the @Response parameter is present, it must have `{ passthrough: true }`.
```

Comment:
Build is failing now (we must escape { } chars)

Reply:
Done